### DISCUSSION TITLE
Portal eventforside opfører sig underligt...

### DISCUSSION THREAD

**Discussion by MartinQwist** [xooom-community/public#17] [Bugs] [answered]
Der er et eller andet i vejen med eventsiden/forsiden på portalen. Jeg oplever flere uregelmæssigheder.

Events sorteres pludseligt ikke kronologisk efter første eventdag: 
![image](https://github.com/user-attachments/assets/b792143e-f914-42b8-8734-9b58c4a5b3b6)

Nogle events bliver uforklarligt off line. Også selvom jeg reloader siden. F.eks. har dette event været væk siden i går (og er nu tilbage igen), efter brugeren satte kapaciteten ned, som ellers ikke forårsager gengodkendelse. Screenshottet er taget på et tidspunkt, hvor eventet var off line. Som man kan se er det godkendet og udgivet, og burde derfor være online: 
![image](https://github.com/user-attachments/assets/79f458e4-ab64-4c97-80db-0dc13be8e64d). 

Kan I forklare det? 

Note til senere: Af flere årsager kunne det være godt, hvis man som frivilligorg havde en side/fane der viste events, som arrangører har oprettet, men som ikke er online.

**Comment by henrikbak** [comment, marked as answer]
Hej Martin,

Det kunne se ud til at sorteringen ikke er rigtig, når man vælger en "Fra-dato" i datofilteret. Så første gang du kommer ind på forsiden er events sorteret kronologisk, men når man filtrerer på dato er sorteringen en anden. Det opretter jeg lige en opgave på.

Når du siger at eventet er offline, vil det så sige at du ikke kan se det i listen over events eller kan du heller ikke tilgå det ved at gå ind på den direkte URL: https://rehelper.dk/portal/kulturens/event/you-may-rest-oi30

**Reply by MartinQwist** [reply]
Fint til den første. 

Ift den anden oplevede jeg, at eventet forsvandt fra portalsiden og fra min liste over events inde på Kulturens Frivilliges backend. Jeg tænkte ikke lige på at kigge på arrangørsiden eller direkte URL. Så det ved jeg ikke. Den er der nu igen - den var væk i et døgns tid. Jeg kan prøve, hvis det sker igen. 

Det er lidt følsomt, fordi vi jo ikke har noget overblik over, hvilke events, der ikke er online. Derfor er det ret tilfældigt, når vi opdager det. Jeg har set det før, men ikke reageret før du. Så jeg tror, at der sker et eller andet....
